app/views/layout.jade

```diff
@@ -45,7 +45,10 @@ html(ng-app='nitApp')
 
       #top-bar
         div.middle
-          p Velkommen til Trondheim!
+          span Velkommen til Trondheim!
+          if union
+            p.navbar-right
+              a#logout-button(href="/logout" target="_self") Logg ut
```

Comment:
This is quite hard to read, the text should probably be white :slightly_smiling_face: